agent/command/s3_put.go

```diff
@@ -191,6 +197,15 @@ func (s3pc *s3put) expandParams(conf *internal.TaskConfig) error {
 	} else {
 		s3pc.skipMissing = false
 	}
+
+	if s3pc.NoOverwrite != "" {
+		s3pc.noOverwriteBool, err = strconv.ParseBool(s3pc.NoOverwrite)
+		if err != nil {
+			return errors.WithStack(err)
+		}
+	} else {
+		s3pc.noOverwriteBool = false
```

Comment:
I know you copied the existing optional code but this is implied already so we don't need to specify it

Reply:
good point. removed.